app/assets/stylesheets/modern-business.scss

```diff
@@ -87,30 +106,38 @@ header.carousel .fill#slider-3 {
   font-family: 'Quattrocento', serif;
   font-size: 50px;
   text-align: center;
+  color: #001B4D;
 }
 
 #subtitle-1 {
   font-family: 'Quattrocento', serif;
   font-size: 30px;
   text-align: center;
+  color: #001B4D;
 }
 
 #subtitle-2 {
   font-family: 'Quattrocento', serif;
   font-size: 25px;
   text-align: center;
+  color: #001B4D;
 }
 
 #copyright {
   text-align: center;
+  font-size: 15px;
 }
 
-#candidate-front-title {
+#candidate-front-title a {
+  font-family: 'Quattrocento', serif;
   text-align: center;
+  color: #001B4D;
```

Comment:
Color literals like `#001B4D` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#001B4D` should be written as `#001b4d`